patcher/src/components/Sound/index.tsx

```diff
@@ -39,40 +40,52 @@ const sounds = {
 export class Sound extends React.Component<SoundProps, SoundState> {
 
   private bgRef: HTMLAudioElement = null;
+  private evhs: any[] = [];       // event handlers
   private audioRefs: { [id: string]: HTMLAudioElement } = {};
 
   constructor(props: SoundProps) {
     super(props);
     this.state = {
       sounds: {},
+      paused: false,
     };
   }
 
   public render() {
     return (
       <div>
-        <audio src='sounds/patcher-theme.ogg' ref={r => this.bgRef = r }/>
-        {this.renderAudioElements()}     
+        <audio loop={true}
+               src='sounds/patcher-theme.ogg'
+               ref={r => this.bgRef = r }/>
+        {this.renderAudioElements()}
       </div>
     );
   }
 
+  private setStateAsync = (sparseState: any) => {
+    setTimeout(() => { this.setState(sparseState); },0);
```

Comment:
I think the solution should be that PatchButtons render function does a timeout on the call from there.